util.py

```diff
@@ -71,32 +72,32 @@ async def prompt_for_turn(
 	TODO: prompt user for what type of turn
 	"""
 
-	menu_items = []
-	menu_items.append(battlecontext.pokemon.Name)
-	menu_items.append(
-		f"{battlecontext.pokemon.CurrentHP} HP {taggify(type_to_string(battlecontext.pokemon.Type))} {taggify(status_to_string(battlecontext.pokemon.StatusEffects))}"
-	)
+	title = battlecontext.pokemon.Name
+	description = f"{battlecontext.pokemon.CurrentHP} HP {taggify(type_to_string(battlecontext.pokemon.Type))} {taggify(status_to_string(battlecontext.pokemon.StatusEffects))}"
 
+	menu_items = []
 	for i, move in enumerate(battlecontext.pokemon.Moves):
-		menu_items.append(f"{RESPONSE_REACTIONS[i]}: {move['Name']}")
-		menu_items.append(
-			f"{taggify(type_to_string(move['Type']))} {move['CurrentPP']}/{move['MaxPP']}"
+		menu_tuple = tuple(
+			(
+				f"{RESPONSE_REACTIONS[i]}: {move['Name']}",
```

Comment:
`RESPONSE_REACTIONS[i]` should be added by the menu, we shouldn't have to care about it outside the menu